WordPress/Classes/ViewRelated/Blog/BlogDetailsSectionFooterView.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+
+class BlogDetailsSectionFooterView: UITableViewHeaderFooterView {
+    private let titleLabel: UILabel = {
+        let titleLabel = UILabel()
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.numberOfLines = 0
+        titleLabel.font = WPStyleGuide.tableviewSectionFooterFont()
+        titleLabel.textColor = WPStyleGuide.greyDarken10()
+        return titleLabel
+    }()
+    private let spacerView: UIView = UIView(frame: .zero)
```

Comment:
You can omit the `: UIView` and let the compiler infer the type. I used to define the type every time but compact code is even better 😊
```suggestion
    private let spacerView = UIView(frame: .zero)
```